enterprise/cmd/frontend/db/db_test.go

```diff
@@ -1,7 +1,20 @@
 package db
 
-import "github.com/sourcegraph/sourcegraph/internal/db/dbtesting"
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sourcegraph/sourcegraph/internal/db/dbtesting"
+)
 
 func init() {
 	dbtesting.DBNameSuffix = "enterprisedb"
 }
+
+func equal(t testing.TB, name string, want, have interface{}) {
+	t.Helper()
+	if !reflect.DeepEqual(want, have) {
```

Comment:
We could use `cmp.Diff` here too?

```suggestion
	if diff := cmp.Diff(have, want, cmpopts.EquateEmpty()); diff != "" {
	    t.Fatalf("%q: %s", name, diff)
	}
```